src/openrct2-ui/windows/NewRide.cpp

```diff
@@ -951,42 +951,39 @@ static void window_new_ride_paint_ride_information(rct_window *w, rct_drawpixeli
     gfx_draw_string_left_wrapped(dpi, gCommonFormatArgs, x, y, width, STR_NEW_RIDE_NAME_AND_DESCRIPTION, COLOUR_BLACK);
 
     // Number of designs available
```

Comment:
This comment can be removed, the code is clear enough and it isn't in the right place any more.

Reply:
Fixed.